### PR TITLE
Navier miniapp 

### PR DESCRIPTION
**Pull request by andersonw1**
This PR compiles PyMFEM with the navier miniapp by including the interface file `navier_solver.i`. The input file `navier_mms.cpp` is also converted to python as an example.

### CODE REVIEW

**Comment by chldkdtn** [comment]
@sshiraiwa if you can review and merge this to master branch, that would be wonderful.

**Comment by sshiraiwa** [comment]
@chldkdtn , @andersonw1, Thank you for contribution. The CI error doesn't seems caused by this PR itself. I am looking into it.

**Comment by sshiraiwa** [comment]
Please let us know if the suggested change is okay for you. If it is okay, we will merge it.  Thank you for fantastic contribution!

File: mfem/_par/navier_solver.i
[the remark this answers is not in the document]
Comment (andersonw1) [reply mfem/_par/navier_solver.i:1]: Thanks for the comment! I just added the three lines you suggested

**Pull request merged**